src/config/dynamic-theme-fixes.config

```diff
@@ -8764,6 +8764,18 @@ section[class$="container"][style^="background-image"] {
 
 ================================
 
+mdigi.tools
+
+CSS
+.color-shade {
+    --darkreader-inline-bgcolor: none;
+}
```

Comment:
Not needed

Reply:
> Not needed

how would i get the colors do not have their saturation inverted then, for [Color Shades](https://mdigi.tools/color-shades/) specifically